tests/unit/-debug/type-test.js

```diff
@@ -437,8 +437,8 @@ test('typed value does not trigger mandatory setter', function(assert) {
   assert.expect(3);
 
   class Foo extends EmberObject {
-    constructor() {
-      super();
+    init() {
```

Comment:
`addObserver` does not work in the constructor in Ember `3.6.0`.

Normally I wouldn't want to change a test to make it pass, but in this case we're only setting up the observer as a means to validate the setter worked correctly, so it seems safe to modify because the signal we receive is the same.